content/about/about_people.rst

```diff
@@ -296,3 +296,38 @@ runnerboi
 .. raw:: html
 
   <br/>
+
+`Catherine Stevens`_
+--------------------
+
+.. image:: /images/cstevens.jpeg
+    :width: 110px
+    :align: right
+    :alt: Catherine Stevens
+
+.. class:: no-breaks
+
+  **Student Systems Engineer**
+
+Catherine joined the OSL in February 2019 and is studying computer science. 
+She previously did UX development for websites, and joined the lab to gain a better 
+understanding of open source technology and how software is written and tested.
+She is interested in how computer science, data analysis and security can be used to 
```

Comment:
> She is interested in how computer science, data analysis and security

Police here. Madam, where's your Oxford comma? >-|

```suggestion
She is interested in how computer science, data analysis, and security can be used to 
```

Reply:
@csrs any thoughts on this?